.github/workflows/autoupdate.yml

```diff
@@ -0,0 +1,61 @@
+name: Autoupdate
+
+on:
+  schedule:
+    - cron: "0 */3 * * *"
+  workflow_dispatch:
+
+
+jobs:
+  autoupdate:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install dependencies
+      run: |
+        sudo pip3 install setuptools wheel
+        sudo pip3 install yq
+        pip3 install -r scripts/requirements.txt
+    - name: Run autoupdate
+      run: |
+        git config user.email "actions@github.com"
+        git config user.name "Github Actions"
+        git fetch origin
+        
+        for file in manifests/*
+        do
+          git reset --hard master
+          ./scripts/autoupdate.py "$file"
+          
+          if [[ `git status --porcelain` ]]; then
+            echo "Things have changed, attempting to create a PR"
+            git status
+            
+            version=$(cat "$file" | yq -r ".version") # "1.5.3"
+            filename=${file##*/} # "Plugin Name.yml"
+            name=${filename%.*} # "Plugin Name"
+            branchname=$(echo "$name-$version" | sed 's/[^a-zA-Z0-9]/_/g') # "Plugin_Name-1_5_3"
+            
+            if git branch -al | grep -F "remotes/origin/$branchname" > /dev/null; then
+              echo "Branch $branchname already exists, discarding changes"
+              git restore manifests/
+
+            else
```

Comment:
What about multiple releases without an update? Release - oops, fix bug - release again, all before the previous is merged?
Basically thinking like dependabot behavior.

Reply:
The branches are version-specific, so it would push to a new branch and open a new PR. This condition is to prevent possible checkout failures in case the workflow runs again while the PR isn't merged yet. In this case it would update the file on master, then try to checkout the branch where this exact change is already committed.

Reply:
Yeah, I'm suggesting the intermediate PR gets closed and its branch deleted.

No point having a PR for version 1.0.0 and 1.0.1 both open.

Of course, having said that, there is a case to be made for keeping both: if we implement plugin-game validation, it'd be possible for the earlier PR to pass the checks, but the second PR to fail the checks. In that case we wouldn't want to close the "intermediate" PR because the latest version introduces a (presumably different) bug.

Reply:
My first thought was "that's way too complicated to be worth the effort" but thinking about it... a simple `gh pr close -d "$old_branch"` should do it.

So the argument is your concerns about specific versions breaking, versus having to manually close PRs once in a blue moon.

(Regarding deleting branches, i plan to follow up with an action that gets rid of all `auto/` branches once their PR is closed)